### PR TITLE
TSE Review 01.

### PR DESCRIPTION
**Pull request by carlosveigadev**
- Added linter files;
- Added gitignore files;
- Added the main files (index, and styles);
- Created the main index code and added the pictures;
- Styled the elements accordingly to the website cloned;
- Updated and fixed the readme file.
- Fixed linter issues using local linter

### CODE REVIEW

**Comment by carlosveigadev** [comment]
I am getting the following error at the GitHub check for linter errors:

_Error: ENOENT: no such file or directory, open '/src/src/stylelint.config.js'_

So I locally fixed the linter errors.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hi @wrakc, you have done a very commendable job on this project! :clap: You have met most of the requirements. However, there are a few issues you need to fix to have this project approved.
These are;
- [x] Your clone appears to be broken on large screens. ([screenshot](https://github.com/trekab/images/blob/master/appleclone.png?raw=true) from my screen). 
- [x] You have to resolve the stickler error you are facing.

__Kindly work on the highlighted issues and submit another code review request.__
_Feel free to contact me on slack for any assistance regarding this review_ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by carlosveigadev** [comment]
Thanks, I will make the changes as suggested!

**Comment by carlosveigadev** [comment]
I fixed the issues with the stickler clone and fixed the margins of the background image when appearing in high-resolution screens, also closed the pull request that I created and reopened this one.

File: assets/styles.css [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: 'Gothic A1', sans-serif;
+}
+
+/* responsiveness */
+
+.resize {
+  min-width: 1440px;
```
Comment (rammazzoti2000) [inline assets/styles.css]: Causing horizontal overflow, change it to max-width instead to fix the issue

File: assets/styles.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+/* Images */
+
+.images {
+  display: flex;
+  justify-content: center;
+  width: auto;
+}
+
+.images img {
+  width: 358px;
```
Comment (rammazzoti2000) [inline assets/styles.css:150]: last step to fix horizontal overflow is to set this width to 99%

File: assets/styles.css [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+input:focus {
+  background: url("../assets/images/lupa_selected.png") no-repeat 1%, white;
+  width: 170px;
+}
+
+/* Text article */
+
+.text {
+  display: flex;
```
Comment (rammazzoti2000) [inline assets/styles.css:109]: CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      align-items: center;
   ```

File: assets/styles.css [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+nav a:hover {
+  background: linear-gradient(0deg, rgba(95, 95, 95, 1) 0%, rgba(65, 65, 65, 1) 50%, rgba(75, 75, 75, 1) 51%, rgba(95, 95, 95, 1) 100%);
+}
+
+nav a:first-child {
+  line-height: 40px;
+  border-radius: 5px 0 0 5px;
+}
+
+.input-div {
```
Comment (rammazzoti2000) [inline assets/styles.css:78]: CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      align-items: center;
   ```

File: assets/styles.css [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+.images img {
+  width: 358px;
+  height: auto;
+  margin: 1px 1px 1px 1px;
+}
+
+/* footer */
+
+footer {
```
Comment (rammazzoti2000) [inline assets/styles.css:157]: CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      justify-content: center;
   ```

File: assets/styles.css [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+  margin: 15px 20px 10px 0;
+  min-width: 580px;
+  color: rgba(0, 0, 0, 0.8);
+}
+
+.useful_links {
+  margin-top: 15px;
+}
+
+.useful_links ul {
```
Comment (rammazzoti2000) [inline assets/styles.css:176]: CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      justify-content: center;
   ```

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @wrakc :wave: great work this far :rocket: 
You used HTML semantic in your project, as well  CSS background and gradients, great job on that.
Also the Readme files is descriptive, Pull Request as well, commits short and meaningful, overall the project looks professional.
There are still some issues left before approving this project, check the comments below :point_down: 
### :stop_sign: Required Changes :construction: :construction_worker: 
- [x] Your project still has some horizontal overflow left, please see the inline comments for the fix
- [x]   CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
   *  No repeating rule, see the inline comments for more details
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### :stop_sign: Status: Required Changes

**Comment by carlosveigadev** [comment]
I fixed the horizontal overflow left according to the suggestion;
I fixed the CSS bad practices according to the inline comments.

**Comment by carlosveigadev** [comment]
- I fixed the horizontal overflow left according to the suggestion;
- I fixed the CSS bad practices according to the inline comments;
- Fixed some linter errors.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks:  Keep it up.
##
### The [clone](https://raw.githack.com/wrakc/apple-clone/feature/index.html) shows good understanding and usage of building with backgrounds and gradients. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
-  The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @wrakc! :+1:
- You can now merge to master and I wish you all the very best in your next tasks 🥇
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**